test/test_fori_loop_with_while_loop_simple_add_dispatch_in_torch.py

```diff
@@ -0,0 +1,38 @@
+import os
+import unittest
+from typing import Callable, Dict, List
+
+import torch
+import torch_xla
+import torch_xla.experimental.fori_loop
```

Comment:
@ManfeiBai can you leave a brief comment saying that "We need to import the underlying implementation function to register with the dispatcher"

Reply:
Thanks, updated in the newest commit